src/Application/CacheableResponseMiddleware.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace MatchBot\Application;
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+/**
+ * Add to routes we want to cache for (currently) up 1 minute each.
+ */
+class CacheableResponseMiddleware implements MiddlewareInterface
+{
+    private const int CACHE_SECONDS = 60; // 1 minute
+
+    #[\Override]
+    public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
+    {
+        $response = $handler->handle($request);
+
+        return $response
+            ->withHeader('Cache-Control', 'public, max-age=' . self::CACHE_SECONDS)
+            ->withHeader('Expires', gmdate('D, d M Y H:i:s', time() + self::CACHE_SECONDS) . ' GMT');
```

Comment:
Should be redundant with the above `Cache-Control` according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Expires but I think it's fairly common to do both, perhaps because for very old HTTP/1.0 clients `Expires` could be the only effective one? https://stackoverflow.com/questions/5799906/what-s-the-difference-between-expires-and-cache-control-headers

But the only client whose caching is very important here is CloudFront itself. Happy to simplify if we think it's better to only keep `Cache-Control`.

Reply:
I don't think it matters having both.